source/read/specify-a-query.txt

```diff
@@ -0,0 +1,264 @@
+      {"_id":3,"name":"oranges","qty":6,"rating":2,"type":["naval","mandarin"]}
+
+Element Operators
+-----------------
+
+Element operators query data based on the presence or type of a field.
+
+To learn more about element operators, see the :manual:`Element Query Operators
+</reference/operator/query-element/>` guide in the {+mdb-server+} manual.
+
+The following example specifies an element operator in a query filter as a
+parameter to the ``MongoDB\Collection::find()`` method. The code returns all
+documents that have a ``color`` field:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/read/specify-queries.php
+      :start-after: start-find-element
+      :end-before: end-find-element
+      :language: php
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      {"_id":1,"name":"apples","qty":5,"rating":3,"color":"red","type":["fuji","honeycrisp"]}
+      {"_id":2,"name":"bananas","qty":7,"rating":4,"color":"yellow","type":["cavendish"]}
+      {"_id":4,"name":"pineapples","qty":3,"rating":5,"color":"yellow"}
+
+Evaluation Operators
+--------------------
+
+Evaluation operators return data based on evaluations of either individual
+fields or the entire collection's documents.
+
+The following list describes common evaluation operators:
+
+- ``$text``: Performs a text search on the documents
+- ``$regex``: Returns documents that match a specified regular expression
+- ``$mod``: Performs a modulo operation on the value of a field and
+  returns documents where the remainder is a specified value
+
+To view a full list of evaluation operators, see the :manual:`Evaluation Query Operators
+</reference/operator/query-evaluation/>` guide in the {+mdb-server+} manual.
+
+The following example specifies an evaluation operator in a query filter as a
+parameter to the ``MongoDB\Collection::find()`` method. The code uses a regular
+expression to return all documents in which the ``name`` field value has at least
+two consecutive ``'p'`` characters:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/read/specify-queries.php
+      :start-after: start-find-evaluation
+      :end-before: end-find-evaluation
+      :language: php
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      {"_id":1,"name":"apples","qty":5,"rating":3,"color":"red","type":["fuji","honeycrisp"]}
+      {"_id":4,"name":"pineapples","qty":3,"rating":5,"color":"yellow"}
+
+Additional Information
+----------------------
+
+To learn more about querying documents, see the :manual:`Query Documents
+</tutorial/query-documents/>` guide in the {+mdb-server+} manual.
+
+.. TODO:
+ To learn more about retrieving documents with the {+php-library+}, see the
+ :ref:`php-retrieve` guide.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods or types discussed in this
+guide, see the following API documentation:
+
+- `MongoDB\\Collection::find() <{+api+}/method/MongoDBCollection-find/>`__
```

Comment:
S: include other types/methods used in the page, such as `insertMany()`, `cursor`?